packages/foam-vscode/src/features/tag-completion.ts

```diff
@@ -5,7 +5,7 @@ import { FoamFeature } from '../types';
 import { mdDocSelector } from '../utils';
 import { SECTION_REGEX } from './link-completion';
 
-export const TAG_REGEX = /#(.*)/;
+export const TAG_REGEX = /#(\S+)/;
```

Comment:
in `hashtag.ts` there is the actual regex used to parse hashtags, could you try using that? Would love to consolidate the two

Reply:
updated in [7695819](https://github.com/foambubble/foam/pull/1183/commits/7695819a2ddc75093801980cb337a4143bc50bba)